Destination/AbstractDestination.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Zenstruck\BackupBundle\Destination;
+
+use Psr\Log\LoggerInterface;
+use Zenstruck\BackupBundle\Rotator\Rotator;
+
+abstract class AbstractDestination implements Destination
```

Comment:
I try not using `Abstract` to prefix abstract classes if I can avoid it. Can we call this something else?

What about an abstract class called `RotatingDestination` instead of the interface? (not sure if I like not using an interface though...)